package.json

```diff
@@ -49,6 +51,8 @@
     "predist": "npm run eslint",
     "prepack": "fixpack && npm run dist",
     "pretest": "npm run dist && npm run build-test-addon-xpi",
-    "test": "export FIREFOX_BINARY=firefox && XPI_NAME=test-addon/test-addon.xpi mocha test"
+    "test": "export FIREFOX_BINARY=firefox && XPI_NAME=test-addon/test-addon.xpi mocha test",
+    "predocformat": "doctoc --title '**Contents**' docs/*.md",
```

Comment:
Doesn't look like `npm run predocformat` was called before pushing PR? I ran it on your PR and got a diff.

```bash
bdanforth ~/projects/shield-studies-addon-utils ((HEAD detached at motin/basic-doc-improvements)) $ npm run predocformat

> shield-studies-addon-utils@4.1.0 predocformat /Users/bdanforth/Projects/shield-studies-addon-utils
> doctoc --title '**Contents**' docs/*.md


DocToccing single file "docs/engineering.md" for github.com.

==================

"docs/engineering.md" will be updated

Everything is OK.

DocToccing single file "docs/survival.md" for github.com.

==================

"docs/survival.md" will be updated

Everything is OK.

DocToccing single file "docs/tutorial.md" for github.com.

==================

"docs/tutorial.md" will be updated

Everything is OK.
```

Reply:
The diff caused by predocformat is reverted by docformat. Basically prettier and doctoc have different ideas of how to format bullet lists in markdown. Removed predocformat and made  npm run docformat will run the two in sequence instead to avoid confusion.